src/blurredLocationDisplay.js

```diff
@@ -39,10 +39,35 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
     return blurredLocations;
   }
 
+  function exampleMarkers() {
+
+    function marker_display() {
+      
+      console.log(options.blurredLocation.getPrecision());
+      var markers = {
+        marker_1: [1.5, 10.7, 0],
```

Comment:
Ah, actually these could stay in the example index.html code -- we just need to display them only at a range of zoom levels corresponding to their precision. So if they're `x.x` precision, we can use [this guide](https://github.com/publiclab/leaflet-blurred-location#precision-and-zoom) to decide what zoom levels they should be shown for!

Reply:
For example, if we're zoomed in so that the grid squares represent `x.xx` precision, we would NOT show markers with only `x.x` precision. But we WOULD show markers with `x.xx` precision or better. Make sense?

Reply:
We will have to bind them to the setZoom function right? Also where do you think we should store the markers, as once we need their objects to remove them. And we will have to remove them once the user zooms in to a greater precision.

Reply:
@jywarren Any comments?

Reply:
Hi, @mridulnagpal -- yes, we should create a listener on `setZoom`, and I think we should *hide* them, but not delete them. Does that make sense?

Reply:
@jywarren Yes, got it!!

Reply:
@jywarren Just checked the hide option for layers is under construction, for now we can keep an array of location and when setZoom is called we remove all the markers and add the  markers with correct precision, sounds good?